HelloWorld/force-app/main/default/lwc/helloWorld/helloWorld.js-meta.xml

```diff
@@ -1,5 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <LightningComponentBundle xmlns="http://soap.sforce.com/2006/04/metadata">
-    <apiVersion>49.0</apiVersion>
-    <isExposed>false</isExposed>
+    <apiVersion>48.0</apiVersion>
+    <isExposed>true</isExposed>
+    <targets>
+        <!-- For further detail refer to https://sfdc.co/b2QmxE -->
+        <target>lightning__AppPage</target>
+        <target>lightning__RecordPage</target>
+        <target>lightning__HomePage</target>
```

Comment:
These are the common targets that you'll see in many of the LWC trailhead modules so I'm using the same ones. With these targets and with `isExposed=true` now the HelloWorld component can be deployed to an org.